c++/include/cps/CondensedOrthogonalMatrix.h

```diff
@@ -139,7 +139,19 @@ namespace cps
     if (ptranspose_)
       const_cast<Eigen::MatrixBase<Derived>&>(M) = transpositions_*M;
     else
+    {
+#if EIGEN_VERSION_AT_LEAST(3, 2, 90)
```

Comment:
Just to make sure: your title and the link to SO mention Eigen 3.3, why doesn't your test starts at 3.3.0 ?

Reply:
`3.2.90` is the beta release of `3.3` (and the default Eigen version on 16.04) and it is also affected by this issue